### ISSUE TITLE
Font Examples Fail

### ISSUE THREAD

**Issue by LennyPhoenix** [pyglet/pyglet#495]
`text/advanced_font.py` fails with the following:
```py
/home/lenny/Source/_Python/pyglet/pyglet/font/freetype.py:168: UserWarning: The current font render does not support st
retching.
  warnings.warn("The current font render does not support stretching.")
Traceback (most recent call last):
  File "/home/lenny/Source/_Python/pyglet/examples/text/advanced_font.py", line 35, in <module>
    image = font.render_to_image("I am rendered as a texture! 🌎", 100, 300)
AttributeError: 'FreeTypeFont' object has no attribute 'render_to_image'
```
This seems like it is likely a linux-specific issue as the `render_to_image` method is implemented in the DirectWrite renderer.

`text/font_comparison.py` fails with the following:
```py
Traceback (most recent call last):
  File "/home/lenny/Source/_Python/pyglet/examples/text/font_comparison.py", line 52, in <module>
    import pyglet.font.win32query as wq
  File "/home/lenny/Source/_Python/pyglet/pyglet/font/win32query.py", line 157, in <module>
    user32 = ctypes.windll.user32
AttributeError: module 'ctypes' has no attribute 'windll'
```
This seems to be a general non-windows issue, as the example tries to import Win32 libraries by default without checking the OS.

**Comment by LennyPhoenix** [comment]
Yes that would certainly be more elegant, I don't currently have the knowledge to implement the required features in FreeType so that might be the best solution for now (unless someone else more knowledgeable wants to help).

> Both examples do state that they are Windows only in the files.

I completely missed that in the docstring, my mistake :sweat_smile:

**Comment by benmoran56** [comment]
We have this check at the top of some of the other examples, so this is probably good to add:
```py
if pyglet.compat_platform != 'win32':
    print("This example is only for Windows")
    exit()
```

It would be great if someone wanted to work on the text code, but I probably wouldn't look at it myself until after 2.0/2.1 is out.

**Comment by caffeinepills** [comment]
This is fixed in master and 1.5 maintenance branch now.
